examples/features/inference_mnist_pytorch/distributed_inference.yaml

```diff
@@ -0,0 +1,30 @@
+name: inference_metrics_example
+entrypoint: >-
+  python3 -m determined.launch.torch_distributed
+  python3 inference.py
+
+resources:
+  slots_per_trial: 2
+
+searcher:
+  name: grid
+  metric: x
+  max_length: 100
+hyperparameters:
+  # Change this to your model name.
+  model_name: mnist_models
+  # Change this to point to each of the model versions you would like to run
```

Comment:
Can you emphasize that this has to be the model version number. I tried using my model version name, e.g. `v1` and it errored.